service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -95,4 +95,28 @@ public int updateLastRunWithRunSet(RunSet runSet) {
                 "run_set_id", runSet.runSetId(),
                 "method_id", runSet.methodVersion().method().methodId())));
   }
+
+  public int getNumberOfMethods(
```

Comment:
I would maybe call this `countMethods` to be closer to the SQL action in question